src/shared/types/solid.d.ts

```diff
@@ -26,6 +26,13 @@ export type AnimationSettings = { duration?: number; delay?: number; timing?: st
 export type TextAlign = 'left' | 'center' | 'right';
 export type ContentAlign = 'center' | 'flexStart' | 'flexEnd' | 'spaceBetween' | 'spaceEvenly';
 export type States = 'active' | 'focus' | 'disabled';
+export type Size = {
```

Comment:
this should maybe live in the key styles? either that or the name should be a bit more explicit